pyinstaller.py

```diff
@@ -165,6 +165,34 @@
         "network_modeling/configs/templates/1.5/dellmellanox/full/*.j2",
         "network_modeling/configs/templates/1.5/dellmellanox/full",
     ),
+    (
```

Comment:
my nit here is that this should really go into the other pr for 1.6 templates ;)